### PR TITLE
KS7 Marshal Overhaul Update

### PR DESCRIPTION
**Pull request by YourPalHolidays**
:cl: YourPalHolidays(Tsar)
rscadd: Added new Marshal gear and custom sprites. (Original credit goes to Eluxor)
rscadd: Overhauls KS7's police department.
tweak: Fixes the GCPD slowdown gear bug.
/:cl:

### CODE REVIEW

**Review by Aroliacue — APPROVED** [review]
I love this. Sprites look smooth!

File: .gitignore [hunk 1]
```diff
@@ -30,3 +30,5 @@ __pycache__
 code/modules/halo/flood/types/flood_shield.dm
 code/modules/halo/flood/types/sangheili.dm
 code/modules/halo/vehicles/types/phantom_dropship.dm
+baystation12.dme
+baystation12.dme
```
Comment (X0-11) [inline .gitignore:34]: why has this been changed

File: baystation12.dme [hunk 2]
```diff
@@ -3350,7 +3351,9 @@
 #include "maps\_gamemodes\_popbalance.dm"
 #include "maps\_gamemodes\gamemode_overrides.dm"
 #include "maps\_gamemodes\randantag\randantag.dm"
+#include "maps\faction_bases\civ_shuttle.dm"
 #include "maps\geminus_city\geminus_city.dm"
+#include "maps\ks7_elmsville\includes.dm"
```
Comment (X0-11) [inline baystation12.dme:3356]: this should not have been PR'd

**Review by X0-11 — CHANGES_REQUESTED** [review]
can you please also put some description of the changes to the PD

File: baystation12.dme [hunk 3]
```diff
@@ -3350,7 +3351,9 @@
 #include "maps\_gamemodes\_popbalance.dm"
 #include "maps\_gamemodes\gamemode_overrides.dm"
 #include "maps\_gamemodes\randantag\randantag.dm"
+#include "maps\faction_bases\civ_shuttle.dm"
```
Comment (X0-11) [inline baystation12.dme:3354]: nor should have this been PR'd

File: .gitignore (hunk 1 above)
Comment (YourPalHolidays) [reply .gitignore:34]: Kerfuffle with inclusion and a ignore githhub that I must have did wrong, there should be no changes to the indicated files, if there is then something is wrong on my end.

File: baystation12.dme (hunk 3 above)
Comment (YourPalHolidays) [reply baystation12.dme:3354]: Gitignore moment.

File: baystation12.dme (hunk 2 above)
Comment (YourPalHolidays) [reply baystation12.dme:3356]: Gitignore moment 2.0.

**Review by YourPalHolidays — COMMENTED** [review]
This update overhauls the KS7 Marshals with new equipment, gear and apparel, as well as changes to their PD including a left that leads to a underground garage, several QoL rooms, a mayors office on the second floor along with a expanded armory and other various facilities to promote a more enjoyable experience for KS7 Colonial Marshal players.

File: .gitignore (hunk 1 above)
Comment (X0-11) [reply .gitignore:34]: there are changes, that's what github tracks

Comment (YourPalHolidays) [reply .gitignore:34]: If that is the case then I will cancel this merge request, I have not knowingly modified those files in any way and look into my own fork to see why those files have been changed.

**Pull request closed without merging**